client/components/report/Chart.tsx

```
@@ -25,7 +33,52 @@ export function Chart({
+              if (!includeEmptyValues[attr]) return false;
+            } else {
+              // 数値
+              const numValue = Number(attrValue);
+              if (isNaN(numValue) || numValue < range[0] || numValue > range[1]) return false;
+            }
+          }
+        }
+        return true;
+      })
+      .map((arg) => arg.arg_id);
+  }, [result.arguments, filterState]);
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)